packages/snap-preact-components/src/components/Atoms/SearchHeader/SearchHeader.tsx

```diff
@@ -139,6 +154,7 @@ export interface SearchHeaderProps extends ComponentProps {
 	subtitleText?: string | ((data: data) => string);
 	correctedQueryText?: string | ((data: data) => string);
 	noResultsText?: string | ((data: data) => string);
+	dymText?: string | ((data: data) => string);
```

Comment:
I realize keeping things short can be helpful to reduce characters and all, but I think we should go with the long variable `didYouMeanText` here to align with the other prop names - eg: `correctedQueryText` (which is even longer).